scss/core/_vars.scss

```diff
@@ -112,3 +120,16 @@ $upsell-padding-y: calc-em(10);
 $upsell-padding-x: calc-em(15);
 $upsell-padding-y-lg: calc-em(10);
 $upsell-padding-x-lg: calc-em(10);
+
+// Breadcrumb
+$breadcrumb-divider: quote("/");
+$breadcrumb-divider-lg: quote('>');
```

Comment:
the use of a different divider across mobile and desktop might be an error from designer, please ask to them in the product-uiux channel. remember that if we can avoid to add additional code/customisations between mobile and desktop is always better, so for any doubt just reach designer any time you need

Reply:
for this moment i keep this one, and asked in the product-uiux channel to Designer, would update once we get confirmation.

Reply:
Confirmed to use same divider `>`